knowledge-base/how-to-display-no-data-message-for-reports-and-subreports.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Display "No Data" message in band report
+description: Show a specific text in case of no data source or an empty one
+type: how-to
+page_title: How to display 'No Data' message for Reports and Subreports
+slug: how-to-display-no-data-message-for-reports-and-subreports
+position: 
+tags: ReportLayout
+ticketid: 1386234
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+</table>
+
+
+## Description
+This KB article gives a brief description of how to show "No Data" message when a report or a subreport has no data source or when the data source is empty.
+
+
+## Solution
+For Report:
+1. Insert a textbox in the Report Header section with value: "No data".
+2. Set the textbox property of Visible to False.
+3. Add a new Conditional Formatting Rule. In the Filters section set the fields as follows:
+- Expression: =Count(1)
```

Comment:
it would be nice if Expression, Operator, and Value are in a code snippet. For example:
```
...
```